@xen-orchestra/proxy/src/app/mixins/remotes.js

```diff
@@ -53,6 +53,10 @@ export default class Remotes {
   @decorateResult(getDebouncedResource)
   @decorateWith(deduped, remote => [remote.url])
   async getHandler(remote) {
+    if (!remote.enabled) {
+      throw new Error('remote is disabled')
+    }
```

Comment:
The proxy has no notion of enabled or disabled remote, this should be handled on xo-server's side.